### PR TITLE
Fix decimals

### PR DESCRIPTION
**Pull request by blasten**
/cc @morethanreal @notwaldorf

### CODE REVIEW

**Comment by cdata** [comment]
Can we add a test?

**Comment by blasten** [comment]
@cdata +2 tests :) PTAL

**Comment by cdata** [comment]
Rock on :+1: :boat:

File: demo/index.html [hunk 1]
```diff
@@ -89,6 +89,10 @@
         <div>B</div>
         <paper-slider class="blue" value="211" max="255" editable></paper-slider>
       </div>
+      <div class="center horizontal layout">
+      <div>O</div>
+      <paper-slider max="1.0" step="0.01" editable></paper-slider>
+      </div>
```
Comment (arthurvr) [inline demo/index.html:95]:
``` html
<div class="center horizontal layout">
  <div>O</div>
  <paper-slider max="1.0" step="0.01" editable></paper-slider>
</div>
```

Comment (blasten) [inline demo/index.html:95]: @arthurevans the `O` meant `Opacity` but I think `&alpha;` is more suitable for the `alpha` channel of a `RGBA` color :)

**Pull request merged**